src/key_value/key_value_client.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2021, USC
+// All rights reserved.
+
+#ifndef SRC_KEY_VALUE_KEY_VALUE_CLIENT_H_
+#define SRC_KEY_VALUE_KEY_VALUE_CLIENT_H_
+
+#include <grpcpp/grpcpp.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "key_value.grpc.pb.h"
+
+
+namespace csci499 {
+using grpc::Channel;
+
+using kvstore::KeyValueStore;
+
+// key value client implementtaion for csci499
+class KeyValueClient {
+ public:
+  explicit KeyValueClient(std::shared_ptr<Channel> channel)
```

Comment:
Add method comments, virtual desctructor.

Reply:
fixed in commits